compatibility/history-proxy/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM node:12.20.1-alpine AS base
+
+WORKDIR /usr/local/app
+
+RUN apk --no-cache add \
+  bash \
+  g++ \
+  ca-certificates \
+  lz4-dev \
+  musl-dev \
+  cyrus-sasl-dev \
+  openssl-dev \
+  make \
+  python3
+
+RUN apk add --no-cache --virtual \
+  .build-deps gcc zlib-dev libc-dev bsd-compat-headers py-setuptools bash
+
+COPY js/package*.json ./
```

Comment:
You are copying package-lock.js, so... 

Remove yarn.lock and add package-lock.js

Reply:
Thanks! I just removed the yarn files and add package-lock.js.